packages/frontend/tests/unit/service/channels/utils/-vote-sorter-test.ts

```diff
@@ -0,0 +1,81 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+import { clearLocalStorage, getStore } from 'emberclear/tests/helpers';
+import { VOTE_ACTION } from 'emberclear/models/vote-chain';
+import { generateSortedVote } from 'emberclear/services/channels/-utils/vote-sorter';
+import {
+  convertObjectToUint8Array,
+  convertUint8ArrayToObject,
+} from 'emberclear/utils/string-encoding';
+import { equalsUint8Array } from 'emberclear/utils/uint8array-equality';
+
+module('Unit | Service | channels/utils/vote-sorter', function (hooks) {
+  setupTest(hooks);
+  clearLocalStorage(hooks);
+
+  module('vote is sorted properly', function () {
+    test('when ran', function (assert) {
+      const store = getStore();
+      let yes1 = store.createRecord('identity', {
```

Comment:
would it be better to do:
```ts
import { buildContact } from 'emberclear/tests/helpers/factories/contact-factory';
// this may need to be modified to add the signing keys
import { setupCurrentUser } from 'emberclear/tests/helpers';

module('Unit ...', function (hooks) {
  setupTest(hooks);
  clearLocalStorage(hooks);
  setupCurrentUser(hooks);
});
// ...

test('when ran', function (assert) {
  let yes1 = buildContact();
  let yes2 = buildContact();
  let no1 = buildContact();
  let no2 = buildContact();
  let remaining1 = buildContact();
  let remaining2 = buildContact();

  let currentUser = getService('current-user').record;

});

```

Reply:
We will probably need to make a buildUser() because they will need to have private signing keys in order to sign previous vote chains.

Reply:
Sounds good 👍